### PR TITLE
Add configure-proxy-registry step in optional operators

### PR DESCRIPTION
**Pull request by samvarankashyap**
The following changes need to happen in order to configure the proxy registry 
and also the pull brew secrets from openshift-config

### CODE REVIEW

**Review by gallettilance — APPROVED** [review]

File: ci-operator/step-registry/optional-operators/configure-proxy-registry/optional-operators-configure-proxy-registry.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```
Comment (stevekuznetsov) [inline ci-operator/step-registry/optional-operators/configure-proxy-registry/optional-operators-configure-proxy-registry.sh:1]: This file needs to be called `optional-operators-configure-proxy-registry-commands.sh`

File: ci-operator/step-registry/optional-operators/configure-proxy-registry/test.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+apiVersion: operator.openshift.io/v1alpha1
```
Comment (stevekuznetsov) [inline ci-operator/step-registry/optional-operators/configure-proxy-registry/test.yaml:1]: I think this will fail validation

**Review by stevekuznetsov — COMMENTED** [review]
Please run `make jobs` and review the shellcheck errors in your script.

File: ci-operator/step-registry/optional-operators/configure-proxy-registry/test.yaml (hunk 2 above)
Comment (samvarankashyap) [reply ci-operator/step-registry/optional-operators/configure-proxy-registry/test.yaml:1]: @stevekuznetsov 
Is it the problem with API version ?
if so which version would you suggest.

**Comment by samvarankashyap** [comment]
@stevekuznetsov Could you PTAL what is going wrong with changes ?

File: ci-operator/step-registry/optional-operators/configure-proxy-registry/test.yaml (hunk 2 above)
Comment (stevekuznetsov) [reply ci-operator/step-registry/optional-operators/configure-proxy-registry/test.yaml]: As in the file is not needed and can be removed.

**Comment by samvarankashyap** [comment]
@stevekuznetsov Could you please take a look at the PR 
Thanks.

**Comment by samvarankashyap** [comment]
@stevekuznetsov bump

**Review by stevekuznetsov — APPROVED** [review]

**Pull request merged**